src/main/java/net/ddellspe/music/bot/listeners/PrefixMessageResponseCommandListener.java

```diff
@@ -35,14 +40,7 @@ public PrefixMessageResponseCommandListener(ApplicationContext applicationContex
                     || command.getFilterChannel(event.getGuildId().get()) == null))
         .filter(
             command ->
-                (manager.getPrefix() + command.getName())
-                    .equals(
-                        event
-                            .getMessage()
-                            .getContent()
-                            .substring(
-                                0,
-                                (manager.getPrefix().length() + command.getName().length() + 1))))
+                event.getMessage().getContent().startsWith(manager.getPrefix() + command.getName()))
```

Comment:
this COULD cause an issue if people say something like `!playing blah` but I kind of don't care about that, I should probably clean this up a bit more, but for now, I'm not going to worry